airbyte-integrations/connectors/source-typeform/source_typeform/spec.json

```diff
@@ -28,5 +28,13 @@
         "uniqueItems": true
       }
     }
+  },
+  "authSpecification": {
+    "auth_type": "oauth2.0",
+    "oauth2Specification": {
+      "rootObject": [],
+      "oauthFlowInitParameters": [],
+      "oauthFlowOutputParameters": [["token"]]
```

Comment:
Is this correctly named as `token` not `access_token` ? because in JAVA part you use `access_token`

Reply:
Nope, plz check 63 row in TypeformOAuthFlow.java